bin/spark-class2.cmd

```diff
@@ -63,7 +63,12 @@ if not "x%JAVA_HOME%"=="x" (
 
 rem The launcher library prints the command to be executed in a single line suitable for being
 rem executed by the batch interpreter. So read all the output of the launcher into a variable.
+:gen
 set LAUNCHER_OUTPUT=%temp%\spark-class-launcher-output-%RANDOM%.txt
+rem SPARK-28302: %RANDOM% would return the same number if we call it instantly after last call,
+rem so we should make sure to generate unique file to avoid process collision of writing into a
```

Comment:
nit.
- `make sure` -> `make it sure`.
- `into a same file` -> `into the same file`.